src/schema/sources.ts

```diff
@@ -472,6 +477,11 @@ export const resolvers: IResolvers<any, Context> = {
       await ensureSourcePermissions(ctx, id);
       return getSourceById(ctx, info, id);
     },
+    sourceHandleTaken: async (_, { handle }: { handle: string }, ctx) => {
```

Comment:
I think we need to lowercase the handle right?

Reply:
I agree, even though we lowercase it in FE, it would be better to ensure it is lowercased here. Thanks!